### ISSUE TITLE
Support editorconfig

### ISSUE THREAD

**Issue by avidseeker** [PyCQA/flake8#1957]
### describe the request

In `.editorconfig`
```dosini
[*]
indent_style = tab
```

In `test.py` use a tab
```py
def main():
	pass
```

Result
```
W191 indentation contains tabs
```

Expected: no warning.

Workaround: add `extend-ignore = W191` to `setup.cfg`.

See https://editorconfig.org/

**Comment by asottile** [comment]
please search the issue tracker next time! there are rejected exact duplicates
